meson.build

```diff
@@ -490,7 +490,7 @@ if sys_windows
     'edje',
     'eet',
     'eeze',
-    'efl',
+    #'efl',
```

Comment:
Why not just removing? same to eo and eolian

Reply:
done: 74aac71